maint/gen_ch4_api.py

```diff
@@ -404,12 +402,8 @@ def dump_stub_file(stub_file, mod, is_inline, is_nm):
             dump_s_param_tail(Out, s, params, ")")
             print("{", file=Out)
             if a['ret'] == 'int':
-                if 'mpidig_params' in a:
-                    s = "    return MPIDIG_%s(" % a['name']
-                    dump_s_param_tail(Out, s, a['mpidig_params'], ");", is_arg=True)
-                else:
-                    print("    int mpi_errno = MPI_SUCCESS;", file=Out)
-                    print("    return mpi_errno;", file=Out)
+                print("    int mpi_errno = MPI_SUCCESS;", file=Out)
+                print("    return mpi_errno;", file=Out)
```

Comment:
Could we do `MPIR_Assert(0)` instead? That way if someone copies the stubs, they can implement in pieces and know what parts are executing as they go.

Reply:
Sure.

Reply:
Added.